cms/static/sass/elements/_modal-window.scss

```diff
@@ -661,12 +661,6 @@
       // give a little space between the sections
       padding-bottom: 10px;
 
-      // indent this group a bit to make it seem like
-      // it is one group, under a header
-      .modal-section-content {
-        margin-left: 25px;
-      }
-
```

Comment:
Actually I added this left indent in as part of our release of Studio. I'm going to put it back in for now.